frigate/config.py

```diff
@@ -264,26 +283,11 @@ class BirdseyeConfig(BaseModel):
 ]
 DETECT_FFMPEG_OUTPUT_ARGS_DEFAULT = ["-f", "rawvideo", "-pix_fmt", "yuv420p"]
 RTMP_FFMPEG_OUTPUT_ARGS_DEFAULT = ["-c", "copy", "-f", "flv"]
-SAVE_CLIPS_FFMPEG_OUTPUT_ARGS_DEFAULT = [
-    "-f",
-    "segment",
-    "-segment_time",
-    "10",
-    "-segment_format",
-    "mp4",
-    "-reset_timestamps",
-    "1",
-    "-strftime",
-    "1",
-    "-c",
-    "copy",
-    "-an",
-]
 RECORD_FFMPEG_OUTPUT_ARGS_DEFAULT = [
     "-f",
     "segment",
     "-segment_time",
-    "60",
+    "10",
```

Comment:
This shouldn't break the media browser in home assistant, but the segments will be 10s clips right? Once we update that to use VOD, it wont matter, so its just an interim issue.

Reply:
Correct. Viewing recordings will be a PITA until updated to use the VOD endpoint. This change WILL break viewing events because the clip won’t exist at the /clips/ endpoint.